Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/HolidayParserChs.cs

```diff
@@ -392,31 +392,31 @@ private static int ConvertChineseToInteger(string yearChsStr)
     internal static class TimexConstants
     {
         internal static readonly DateObject now = DateObject.Now;
-        internal static readonly DateObject yuandan = new DateObject(now.Year, 1, 1);
-        internal static readonly DateObject chsnationalday = new DateObject(now.Year, 10, 1);
-        internal static readonly DateObject laborday = new DateObject(now.Year, 5, 1);
-        internal static readonly DateObject christmasday = new DateObject(now.Year, 12, 25);
-        internal static readonly DateObject loverday = new DateObject(now.Year, 2, 14);
-        internal static readonly DateObject chsmilbuildday = new DateObject(now.Year, 8, 1);
-        internal static readonly DateObject foolday = new DateObject(now.Year, 4, 1);
-        internal static readonly DateObject girlsday = new DateObject(now.Year, 3, 7);
-        internal static readonly DateObject treeplantday = new DateObject(now.Year, 3, 12);
-        internal static readonly DateObject femaleday = new DateObject(now.Year, 3, 8);
-        internal static readonly DateObject childrenday = new DateObject(now.Year, 6, 1);
-        internal static readonly DateObject youthday = new DateObject(now.Year, 5, 4);
-        internal static readonly DateObject teacherday = new DateObject(now.Year, 9, 10);
-        internal static readonly DateObject singlesday = new DateObject(now.Year, 11, 11);
-
-        internal static readonly DateObject halloweenday = new DateObject(now.Year, 10, 31);
-
-        internal static readonly DateObject midautumnday = new DateObject(now.Year, 8, 15);
-        internal static readonly DateObject springday = new DateObject(now.Year, 1, 1);
-        internal static readonly DateObject chuxiday = new DateObject(now.Year, 1, 1).AddDays(-1);
-
-        internal static readonly DateObject lanternday = new DateObject(now.Year, 1, 15);
-        internal static readonly DateObject qingmingday = new DateObject(now.Year, 4, 4);
-        internal static readonly DateObject dragonboatday = new DateObject(now.Year, 5, 5);
-        internal static readonly DateObject chongyangday = new DateObject(now.Year, 9, 9);
+        internal static readonly DateObject yuandan = DateObject.MinValue.SetValue(now.Year, 1, 1);

```

Comment:
We should keep the old creation in constants. Only use your new method in DateObjects that are created from user provided tokens.